src/cli/run-command.c

```diff
@@ -146,7 +146,7 @@ static int finish_command(struct command *cmd)
 
 int run_command(char **argv)
 {
-  struct command cmd = { argv, 0 };
+  struct command cmd = { .argv=argv, .pid=0 };
```

Comment:
Same thing, if you are going to add designations just for the sake of having them, do it separately.

We also surround operators with whitespace (and I would prefer seeing initializers being on their own lines).

Reply:
I added the whitespaces. But I do not add the designations just for the sake of having them, this is actually what I do in this patch, either add designations or init all members explicitly. If we only init several (not all) fields without designations it is going to trigger the warning that I try get rid of with this patch.

Reply:
Yeah, you are absolutely correct. I have no idea why I said that, sorry.